MonolithDev/gettingSongs/database.py

```
@@ -0,0 +1,617 @@
+
+
+def get_engine():
+    global _engine, _SessionLocal
+    if _engine is not None:
+        return _engine
+
+    database_url = get_database_url()
+    if not database_url:
+        raise RuntimeError(
+            f"Database URL not configured. Set the {DATABASE_URL} environment variable."
+        )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)